src/docbuilder/rustwide_builder.rs

```diff
@@ -766,115 +746,6 @@ mod tests {
-                )
-                .unwrap();
-            let row = rows.get(0).unwrap();
-
-            assert!(row.get::<_, bool>("rustdoc_status"));
-            assert_eq!(row.get::<_, String>("default_target"), default_target);
-            assert!(row.get::<_, Option<i32>>("total_items").is_some());
-            assert!(!row.get::<_, bool>("archive_storage"));
-
-            let mut targets: Vec<String> = row
-                .get::<_, Value>("doc_targets")
-                .as_array()
-                .unwrap()
-                .iter()
-                .map(|v| v.as_str().unwrap().to_owned())
-                .collect();
-            targets.sort();
-            assert_eq!(
-                targets,
-                vec![
-                    "i686-pc-windows-msvc",
-                    "i686-unknown-linux-gnu",
-                    "x86_64-apple-darwin",
-                    "x86_64-pc-windows-msvc",
-                    "x86_64-unknown-linux-gnu",
-                ]
-            );
-
-            let storage = env.storage();
-            let web = env.frontend();
-
-            // sources exist
-            assert!(storage.exists(&format!("sources/{}/{}/src/lib.rs", crate_, version))?);
-            assert_success(
-                &format!("/crate/{}/{}/source/src/lib.rs", crate_, version),
-                web,
-            )?;
-
-            let base = format!("rustdoc/{}/{}", crate_, version);
-
-            // default target was built and is accessible
-            assert!(storage.exists(&format!("{}/{}/index.html", base, crate_path))?);
-            assert_success(&format!("/{}/{}/{}", crate_, version, crate_path), web)?;
-
-            // other targets too
-            for target in DEFAULT_TARGETS {
-                let target_docs_present =
-                    storage.exists(&format!("{}/{}/{}/index.html", base, target, crate_path))?;
-
-                let target_url = format!(
-                    "/{}/{}/{}/{}/index.html",
-                    crate_, version, target, crate_path
-                );
-
-                if target == &default_target {
-                    assert!(!target_docs_present);
-                    assert_redirect(
-                        &target_url,
-                        &format!("/{}/{}/{}/index.html", crate_, version, crate_path),
-                        web,
-                    )?;
-                } else {
-                    assert!(target_docs_present);
-                    assert_success(&target_url, web)?;
-                }
-            }
-
-            Ok(())
-        })
-    }
-
-    #[test]
-    #[ignore]
-    fn test_build_crate_into_archive() {
-        wrapper(|env| {
-            let crate_ = DUMMY_CRATE_NAME;
-            let crate_path = crate_.replace("-", "_");
-            let version = DUMMY_CRATE_VERSION;
-            let default_target = "x86_64-unknown-linux-gnu";
-
-            env.override_config(|config| {
-                config.use_archive_storage = true;
```

Comment:
This is not what I meant - now we never test retrieving the old format at all. My suggestion was to make `use_archive_storage` test-only, so that we can keep this test without it affecting the server in prod.